src/ui/components/home/site_summary.rs

```diff
@@ -1,14 +1,53 @@
 use crate::{
   contexts::site_resource_context::SiteResource,
-  i18n::*,
-  ui::components::common::{counts_badge::CountsBadge, unpack::Unpack},
-  utils::derive_query_signal,
+  ui::components::common::{
+    icon::{Icon, IconSize, IconType},
+    unpack::Unpack,
+  },
+  utils::{derive_query_signal, format_number_si},
 };
 use leptos::*;
+use leptos_router::A;
+
+#[component]
+fn UserStatRow(count: i64, text: &'static str) -> impl IntoView {
+  view! {
+    <tr class="*:p-2.5 [&:not(:first-child)]:border-t-2 [&:not(:first-child)]:border-accent">
+      <td class="text-xs font-semibold">{text}</td>
+      <td class="text-center font-bold">{format_number_si(count)}</td>
+    </tr>
+  }
+}
+
+#[component]
+fn AdminCard(
```

Comment:
Up to you of course, but I'd recommend just sticking with a generic `UserListing` component that can be used nearly everywhere. 

Those are nice because its uniform, you only have one to maintain, can create consistent coloring and linking, etc.

Reply:
I'm thinkg of using something like the `PersonListing` from the old UI for showing the user's info on posts, comments, and DMs. This and (possibly, depending on how it looks) the person info on the profile page I'm thinking of making different from the more generic listing.